config/flags_network.go

```diff
@@ -69,6 +69,12 @@ var (
 		Usage: "Enables incoming traffic filtering",
 		Value: false,
 	}
+	// FlagOutgoingFirewall enables outgoing traffic filtering.
+	FlagOutgoingFirewall = cli.BoolFlag{
+		Name:  "outgoing-firewall",
+		Usage: "Enables outgoing traffic filtering",
+		Value: false,
```

Comment:
Currently it's on by default, do we want to change this behaviour to false?

Reply:
Yeah, I want to keep it disabled for now, I suspect it leads to some connectivity fails.